esm-samples/jsapi-angular-cli/src/app/app.component.ts

```diff
@@ -70,31 +70,29 @@ export class AppComponent implements OnInit, OnDestroy {
     return this.view.when();
   }
 
-  // tslint:disable-next-line:use-lifecycle-interface
```

Comment:
Class `AppComponent` implements interface `OnInit`, this comment does not have any effect.